### DISCUSSION TITLE
Welcome to hm2500pub Discussions!

### DISCUSSION THREAD

**Discussion by noone2k** [noone2k/hm2500pub#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mastertape** [comment]
Yeah

**Comment by Fuxifux** [comment]
Sehr interessant, da bin ich gerne dabei !
